RawOpsGenerated.swift

```diff
@@ -35551,20 +36896,690 @@ public static func xdivy<T: FloatingPoint & TensorFlowScalar>(
+}
+
+/// Computes the eigen decomposition of a batch of self-adjoint matrices
+///
+/// (Note: Only real inputs are supported).
+///
+/// Computes the eigenvalues and eigenvectors of the innermost M-by-N matrices in
+/// tensor such that tensor[...,:,:] = u[..., :, :] * Diag(s[..., :]) * Transpose(v[...,:,:]).
+///
+/// - Parameter a: the input tensor.
+///
+/// - Attrs:
+///   - max_iter: maximum number of sweep update, i.e., the whole lower triangular
+///     part or upper triangular part based on parameter lower. Heuristically, it has
+///     been argued that approximatly log(min (M, N)) sweeps are needed in practice
+///     (Ref: Golub & van Loan "Matrix Computation").
+///   - epsilon: the tolerance ratio.
+///   - precision_config: a serialized xla::PrecisionConfig proto.
+///
+/// - Outputs:
+///   - s: Singular values. The values are sorted in reverse order of magnitude, so
+///     s[..., 0] is the largest value, s[..., 1] is the second largest, etc.
+///   - u: Left singular vectors.
+///   - v: Right singular vectors.
+@inlinable @inline(__always)
+public static func xlaSvd<T: Numeric & TensorFlowScalar>(
+  _ a: Tensor<T>,
+  maxIter: Int64,
+  epsilon: Double,
+  precisionConfig: String
+) -> (s: Tensor<T>, u: Tensor<T>, v: Tensor<T>) {
+  let nOutputs = Int(1) + Int(1) + Int(1)
+  let op = makeOp("XlaSvd", nOutputs)
+  op.setAttr("max_iter", maxIter)
+  op.setAttr("epsilon", epsilon)
+  op.setAttr("precision_config", precisionConfig)
+  op.setAttr("T", T.tensorFlowDataType)
+  let _ = op.addInput(a)
+  return op.execute(Int(1), Int(1), Int(1))
+}
+
+/// output = input; While (Cond(output)) { output = Body(output) }
+///
+/// - Parameter input: A list of input tensors whose types are T.
+///
+/// - Attrs:
+///   - cond: A function takes 'input' and returns a tensor.  If the tensor is
+///     a scalar of non-boolean, the scalar is converted to a boolean
+///     according to the following rule: if the scalar is a numerical
+///     value, non-zero means True and zero means False; if the scalar is
+///     a string, non-empty means True and empty means False. If the
+///     tensor is not a scalar, non-emptiness means True and False
+///     otherwise.
+///   - body: A function that takes a list of tensors and returns another
+///     list of tensors. Both lists have the same types as specified by T.
+///
+/// - Output output: A list of output tensors whose types are T.
+@inlinable @inline(__always)
+public static func xlaWhile<
+    T: TensorArrayProtocol,
+    CondIn: TensorGroup,
+    CondOut: TensorGroup,
+    BodyIn: TensorGroup,
+    BodyOut: TensorGroup
+>(
+  _ input: T,
+  cond: (CondIn) -> CondOut,
+  body: (BodyIn) -> BodyOut
+) -> T {
+  let nOutputs = Int(input._typeList.count)
+  let op = makeOp("XlaWhile", nOutputs)
+  op.setAttr("T", input._typeList)
+  op.setAttr("cond", cond)
+  op.setAttr("body", body)
+  let _ = op.addInputList(input)
+  return op.execute(Int(input._typeList.count))
+}
+
```

Comment:
Where does this big addition come from? Changes to `generate_wrappers.py` don't look related, do they?

Reply:
I used a newer version of TensorFlow python library, which presumably has more operations.